package.json

```diff
@@ -29,21 +38,7 @@
     "enzyme": "3.9.0",
     "enzyme-adapter-react-16": "1.10.0",
     "enzyme-to-json": "3.3.5",
-    "eslint": "5.14.1",
```

Comment:
we need `eslint`.

Reply:
I meant remove `eslint-plugin-xxx` plugins that do not need it.  We rely on this now https://github.com/verdaccio/eslint-config-verdaccio.

The `.eslintrc.` config file must looks like

```{
  "extends": [
    "@verdaccio"
  ],
  "rules": {
    "@typescript-eslint/no-var-requires": 0
  }
}
```
https://github.com/verdaccio/verdaccio-google-cloud/blob/master/.eslintrc

Reply:
Juan please I didn't finish...My commits have wip = work in progress